app/controllers/admin/assets_controller.rb

```diff
@@ -3,7 +3,7 @@
 module Admin
   class AssetsController < ApplicationController
     before_action :set_submission
-    before_action :set_asset, only: %i[show destroy]
+    before_action :set_asset, only: %i[show destroy download]
```

Comment:
**question**: Does this new `download` action still exist?

Reply:
no, forgot to remove it, thanks